### PR TITLE
[FIX]Trigger on play even with auto start

### PR DESCRIPTION
**Pull request by Volubyl**
**Detailed purpose of the PR**

Consider a resourced as viewed even after a video autostart.

**Result and observation**

Before

Kapture 2019-12-03 at 15 53 07

After

Kapture 2019-12-03 at 15 58 06

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [X] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "flow": "lerna run flow",
     "codecov": "codecov",
     "publish:beta": "lerna publish --npm-tag beta --exact",
-    "publish:canary": "lerna publish --canary --exact patch --no-git-tag-version --no-push --npm-client npm",
+    "publish:canary": "lerna publish --canary --exact patch --no-git-tag-version --no-push",
```
Comment (Fausto95) [inline package.json]: What's that?

Comment (VincentCATILLON) [reply package.json]: I agree, it should be done in another PR.

FYI: https://github.com/lerna/lerna/blob/master/commands/publish/CHANGELOG.md#372-2018-12-21

Comment (Volubyl) [reply package.json]: If we keep those flags the script pops an error :

Screenshot 2019-12-03 at 16 25 18

Comment (VincentCATILLON) [reply package.json]: Introduced there: https://github.com/CoorpAcademy/components/pull/1638

**Comment by VincentCATILLON** [comment]
I can't approve, because I worked on it, even if I'm ok

**Pull request merged**